packages/@blockprotocol/graph/src/stdlib/subgraph/builder.ts

```diff
@@ -0,0 +1,68 @@
+import { addEntitiesToSubgraphByMutation } from "../../internal/mutate-subgraph.js";
+import {
+  Entity,
+  EntityEditionId,
+  GraphResolveDepths,
+  Subgraph,
+} from "../../types.js";
+
+/**
+ * Builds a Subgraph from a given set of entities, which may include 'link entities' –
+ * i.e. entities that represent relationships between other entities – or other entities.
+ *
+ * The set of entities should represent the result of a query on a graph,
+ * and the 'roots' and 'depths' for that query provided along with the data.
```

Comment:
I don't really understand this sentence. The set of entities represent the roots and depths for ... ?

Reply:
any clearer?
```suggestion
 * The set of entities should represent the result of a query on a graph.
 * The 'roots' and 'depths' used for that query should be provided along with the data.
```

Reply:
Much, thank you